components/builder-admin-proxy/hooks/health_check

```diff
@@ -5,7 +5,8 @@ set -x
 # default return code is 0
 rc=0
 
-hab pkg exec core/curl curl --head --fail --max-time 1 http://localhost
+hab pkg exec core/curl curl --head --fail --max-time 1 \
```

Comment:
We should switch this to use a `{{pkgPathFor …}}` helper as this is an anti-patter we want to avoid. However, let's clean that up afterwards--it was there before, it can be fixed when things cool down a touch.